### PR TITLE
change startup wmclass

### LINKED ISSUE #249: flatpak desktop file
please change the .desktop file StartupWMClass from gittyup to com.github.gittyup.gittyup, this is causing it to display the wrong icon on wayland.
Thanks in advance

### PR DESCRIPTION
**Pull request by Murmele**
fixes #249

- Removing Name[de_DE] leads to use Name for showing the desktop file name. So in the apps overview on a german system `Gittyup` instead of `com.github.Murmele.Gittyup.desktop` is shown

### CODE REVIEW

**Comment by Murmele** [comment]
@jjgalvez can you test this build?

**Comment by jjgalvez** [comment]
@Murmele do you have a flatpak build I can test?

**Comment by Murmele** [comment]
you can find the build artifacts at: https://github.com/Murmele/Gittyup/actions/runs/2903399776

**Comment by jjgalvez** [comment]
@Murmele assuming I did it correctly downloading the flatpak and installing it manually I am still getting generic icons

**Comment by jjgalvez** [comment]
@Murmele the window class is still reporting itself as com.github.gittyup.gittyup

**Comment by Murmele** [comment]
Maybe during flatpak build the desktop file will be updated and therefore the changes are overriden

File: rsrc/linux/com.github.Murmele.Gittyup.desktop [hunk 1]
```diff
@@ -3,10 +3,10 @@ Encoding=UTF-8
...
 Exec=Gittyup
-StartupWMClass=gittyup
+StartupWMClass=com.github.Murmele.Gittyup
 Icon=gittyup
 Terminal=false
 Type=Application
 Categories=Development;
 X-Desktop-File-Install-Version=0.1
-Name[de_DE]=com.github.Murmele.Gittyup.desktop
+Name[de_DE]=Gittyup
```
Comment (Murmele) [inline rsrc/linux/com.github.Murmele.Gittyup.desktop:12]:
```suggestion
```

**Pull request merged**